src/schemas/zodSchema/loginSchema.ts

```diff
@@ -0,0 +1,9 @@
+import { z } from 'zod';
+
+const LoginSchema = z.object({
```

Comment:
If you can also export a type for the schema properties which we can use, something like this:
```
export type LoginSchemaType = z.infer<typeof LoginSchema>
```
It will be cool.

Reply:
I thought you previously said the types won't be necessary?

Reply:
>I think the schema type itself is not important, you can use ZodSchema type for that which is more generic

In your words, or maybe I didn't get what you meant exactly?

Reply:
> > I think the schema type itself is not important, you can use ZodSchema type for that which is more generic
> 
> In your words, or maybe I didn't get what you meant exactly?

Both are not the same, the schema and the schema property types are different.
Doing something like this in the response controller to access the email field will fail with Typescript.
``` req.body as typeof LoginSchema
    req.body.email
```
But this will work
```
req.body as z.infer<typeof LoginSchema>
req.body.email
```

Reply:
Oh, I get your statement now